docs/guide/src/en/interactivity/event_handlers.md

```diff
@@ -51,6 +51,8 @@ Any event handlers will still be called.
 
 > Normally, in React or JavaScript, you'd call "preventDefault" on the event in the callback. Dioxus does _not_ currently support this behavior. Note: this means you cannot conditionally prevent default behavior based on the data in the event.
 
+> Note for web developers: on Javascript the default behavior is to **submit the form**, while on Dioxus the default behavior is **to not submit the form**. Consequently, specifying `prevent_default: "onsubmit"` will cause the form to be submitted.
```

Comment:
Note that this should only prevent the default form behavior if a listener is attached to the element. If there is no listener, the form should act as normal

Reply:
Thank you for noticing and point it out, and sorry for missing it.

I rephrased the note in both languages. Hope it is correct and clear now.

Thanks for reviewing it!